tests/test.liq

```diff
@@ -111,6 +111,51 @@ def test.equals(v, v') =
   end
 end
 
+def test.not_equals(first, second) =
+  if
+    first == second
+  then
+    error.raise(
+      error.failure,
+      "expected\r\n
+      #{string.quote(string(first))} == #{string.quote(string(second))}\r\n
+      to be not equal.\r\n"
+    )
+
+    test.fail()
+  end
+end
+
+def test.almost_equals(~places=7, first, second) =
```

Comment:
I think `digits` would be more clear as it specifically names what is being checked. Some comment too:
```suggestion
# Test float equality
# @param ~digits Test equality up-to the given number of decimal digits
def test.almost_equals(~digits=7, first, second) =
```

Reply:
How about
```ruby
# Compares two float values for approximate equality.
# Values are considered almost equal if their difference
# is within 10^-digits.
# @param ~digits The number of decimal digits to compare. Default is 7.
def test.almost_equals(~digits=7, first, second) =
```

Reply:
Works for me!

Reply:
Done